Makefile

```diff
@@ -10,13 +10,13 @@ install: package.json ## Install dependencies
 	@$(PKG) install
 
 watch: ## continuously compile ES6 files to JS
-	NODE_ENV=development ./node_modules/.bin/webpack --watch
+	@yarn vite build --watch
 
 test: ## Launch unit tests
-	@NODE_ENV=test ./node_modules/.bin/jest
+	@NODE_ENV=test NODE_OPTIONS="$$NODE_OPTIONS --experimental-vm-modules" ./node_modules/.bin/jest
```

Comment:
this I don't understand. Would you ind explaining what it does?

Reply:
Make jest accept our ESM code: https://jestjs.io/docs/ecmascript-modules

Reply:
Does that mean that, in order to use the CLI, our users will have to use the same flag? This would be a BC.

Reply:
Nope, as said in the PR desc, I tested the CLI. I did it without any flag on latest Node LTS

Reply:
The minimum required node version is 16.